src/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 300;
+}
+
+.search-container {
+  display: flex;
+  justify-content: center;
+  align-items: left;
+  margin-top: 10px;
+  margin-left: 12px;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  height: 48px;
+  padding: 0 16px;
+}
+
+input[type="text"] {
+  border: none;
+  flex: 1;
+  font-size: 12px;
+  height: 100%;
+  margin-right: 10px;
+  outline: none;
+}
+
+.search-box {
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  align-items: left;
+  width: 100%;
+  height: 75px;
+  background-repeat: no-repeat;
+  background-size: 16px;
+  padding-right: 1px;
+  border-radius: 10px;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.14);
+  margin-top: 10px;
+  border: 2px solid rgb(228, 223, 223);
+  text-align: left;
```

Comment:
All visual styles need to move to the `input` class name selector. Form may have just `width: 100%`. All other styles should be for the input.